src/interfaces/apv.ts

```diff
@@ -4,3 +4,5 @@ export interface IApv {
   signer: string;
   extra: { [key: string]: string };
 }
+
+export type ISimpleApv = Pick<IApv, "version" | "extra">;
```

Comment:
Could we make `signature` and `signer` optional instead? Or make `IApv` extend `ISimpleApv`?

Reply:
make `IApv` extend `ISimpleApv`!

Reply:
Doooooone